drivers/serial/uart_ite_it8xxx2.c

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2021 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT ite_it8xxx2_uart
+
+#include <device.h>
+#include <drivers/gpio.h>
+#include <drivers/uart.h>
+#include <kernel.h>
+#include <pm/device.h>
+#include <soc.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(uart_ite_it8xxx2, LOG_LEVEL_ERR);
+
+struct uart_it8xxx2_config {
+	uint8_t port;
+	uint8_t wakeup_pin;
+	/* GPIO handle */
+	const struct device *gpio_dev;
+	/* UART handle */
+	const struct device *uart_dev;
+};
+
+enum uart_port_num {
+	UART1 = 1,
+	UART2,
+};
+
+#ifdef CONFIG_PM_DEVICE
+__weak void uart1_wui_isr_late(void) { }
+__weak void uart2_wui_isr_late(void) { }
+
+void uart1_wui_isr(const struct device *gpio, struct gpio_callback *cb,
+		   uint32_t pins)
+{
+	/* Disable interrupts on UART1 RX pin to avoid repeated interrupts. */
+	gpio_pin_interrupt_configure(gpio, (find_msb_set(pins) - 1),
```

Comment:
if return code is to be ignored make it explicit: `(void)func()`.

Reply:
Done.